exotica/doc/index.rst

```diff
@@ -9,16 +9,17 @@ EXOTica
 For home page go
 `here <http://wcms.inf.ed.ac.uk/ipab/slmc/research/EXOTica>`__.
 
+
 Overview
 ========
 
--  `Home <index.html>`__
 -  `Installation <Installation.html>`_
--  `Requirements <Prerequisites.html>`__
--  `Quick Start: Python <Quickstart.html>`__
+-  `Quick Start: Python <Quickstart-python.html>`__
 -  `Quick Start: C++ <Quickstart-cpp.html>`__
 -  `Setting up Problems and Solvers <Setting-up-problems-and-solvers.html>`__
 -  `Using EXOTica <Using-EXOTica.html>`_ 
+
+-  `Requirements <Prerequisites.html>`__
```

Comment:
I'd like this header renamed - Requirements is way too generic and I had to open it to know what it means.